sdk/opa_test.go

```diff
@@ -84,6 +84,18 @@ func TestPlugins(t *testing.T) {
 	defer opa.Stop(ctx)
 }
 
+func TestPluginPanic(t *testing.T) {
+	ctx := context.Background()
+
+	opa, err := sdk.New(ctx, sdk.Options{})
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer opa.Stop(ctx)
```

Comment:
[nit] there's nothing to gain from a `defer` in this position. Let's just call
```suggestion
	opa.Stop(ctx)
```

Reply:
Added that suggestion.